src/ACEtk/block/SecondaryParticleAngularDistributionBlock.hpp

```diff
@@ -0,0 +1,152 @@
+#ifndef NJOY_ACETK_BLOCK_SECONDARYPARTICLEANGULARDISTRIBUTIONBLOCK
+#define NJOY_ACETK_BLOCK_SECONDARYPARTICLEANGULARDISTRIBUTIONBLOCK
+
+// system includes
+#include <variant>
+
+// other includes
+#include "ACEtk/block/details/BaseAngularDistributionBlock.hpp"
+
+namespace njoy {
+namespace ACEtk {
+namespace block {
+
+/**
+ *  @class
+ *  @brief The continuous energy LANDP,ANDP and LANDH,ANDH blocks with the
+ *         angular distribution data for secondary particles that are not the
+ *         incident particle type
+ *
+ *  The SecondaryParticleAngularDistributionBlock class contains angular
+ *  distribution data, one for each the NXS(6) reaction numbers on the MTRP
+ *  block (for the LANDP,ANDP blocks) and each of the NTRO(ITYPE) reactions on
+ *  the MTRH(ITYPE) block. The order of the distribution data sets is the same
+ *  as the order of the reaction numbers in the corresponding MTR block.
+ */
+class SecondaryParticleAngularDistributionBlock :
+    protected details::BaseAngularDistributionBlock {
+
+  /* fields */
+
+  /* auxiliary functions */
+
+public:
+
+  /* type alias */
+  using DistributionData = details::BaseAngularDistributionBlock::DistributionData;
+
+  /* constructor */
+  #include "ACEtk/block/SecondaryParticleAngularDistributionBlock/src/ctor.hpp"
+
+  /* methods */
+
+  /**
+   *  @brief Return the number of reactions that produce the projectile
+   *         (excluding elastic )
```

Comment:
Does elastic make sense in this block?

Reply:
It absolutely does not for incident neutron and incident charged particle ACE files. And in photonuclear it makes no sense to have an elastic so not there either? Probably a leftover from copy/pasting since this block looks like the AND one.

Reply:
It seems this has been corrected already.